moto/ec2/responses/security_groups.py

```diff
@@ -121,22 +131,23 @@ def _process_rules_from_querystring(self) -> Any:
                 prefix_list_ids,
             ) = parse_sg_attributes_from_dict(ip_permission)
 
-            yield (
-                group_name_or_id,
-                ip_protocol,
-                from_port,
-                to_port,
-                ip_ranges,
-                source_groups,
-                prefix_list_ids,
-                security_rule_ids,
-            )
+            yield {
+                "group_name_or_id": group_name_or_id,
+                "ip_protocol": ip_protocol,
+                "from_port": from_port,
+                "to_port": to_port,
+                "ip_ranges": ip_ranges,
+                "sgrule_tags": sg_rule_tags,
+                "source_groups": source_groups,
+                "prefix_list_ids": prefix_list_ids,
+                "security_rule_ids": security_rule_ids,
+            }
 
     def authorize_security_group_egress(self) -> str:
         self.error_on_dryrun()
 
         for args in self._process_rules_from_querystring():
-            rule, group = self.ec2_backend.authorize_security_group_egress(*args)
+            rule, group = self.ec2_backend.authorize_security_group_egress(**args)
```

Comment:
```suggestion
            rule, group = self.ec2_backend.authorize_security_group_egress(**kwargs)
```
I like the change to make them keyword-arguments! Makes it easier to debug. Just a minor nitpick that the name should reflect this.